dvclive/report.py

```diff
@@ -0,0 +1,88 @@
+import argparse
+import json
+from pathlib import Path
+
+from dvc_render.html import render_html
+from dvc_render.image import ImageRenderer
+from dvc_render.vega import VegaRenderer
+
+from dvclive.data import PLOTS, Image, Scalar
+from dvclive.data.plot import Plot
+from dvclive.utils import parse_tsv, to_base64_url
+
+
+def get_scalar_renderers(scalars_folder):
+    renderers = []
+    for suffix in Scalar.suffixes:
+        for file in Path(scalars_folder).rglob(f"*{suffix}"):
+            data = parse_tsv(file)
+            for row in data:
+                row["rev"] = "workspace"
+            name = str(file.relative_to(scalars_folder).with_suffix(""))
+            properties = {"x": "step", "y": name}
+            renderers.append(VegaRenderer(data, name, **properties))
+    return renderers
+
+
+def get_image_renderers(images_folder):
+    renderers = []
+    for suffix in Image.suffixes:
+        for file in Path(images_folder).rglob(f"*{suffix}"):
+            name = str(file.relative_to(images_folder))
+            data = [
+                {
+                    ImageRenderer.SRC_FIELD: to_base64_url(file),
+                    ImageRenderer.TITLE_FIELD: name,
+                }
+            ]
+            renderers.append(ImageRenderer(data, name))
+    return renderers
+
+
+def get_plot_renderers(plots_folder):
+    renderers = []
+    for suffix in Plot.suffixes:
+        for file in Path(plots_folder).rglob(f"*{suffix}"):
+            name = file.stem
+            plot = PLOTS[name]
+            data = json.loads(file.read_text())
+            if name in data:
+                data = data[name]
+            for row in data:
+                row["rev"] = "workspace"
+            plot.dump_template(file.with_suffix(".template"))
+            properties = {"template": file.with_suffix(".template")}
+            renderers.append(VegaRenderer(data, name, **properties))
+    return renderers
+
+
+def html_report(dvclive_folder, output_html_path):
+    dvclive_path = Path(dvclive_folder)
+    renderers = []
+    renderers.extend(get_scalar_renderers(dvclive_path / Scalar.subfolder))
+    renderers.extend(get_image_renderers(dvclive_path / Image.subfolder))
+    renderers.extend(get_plot_renderers(dvclive_path / Plot.subfolder))
+
+    render_html(renderers, output_html_path, refresh_seconds=5)
+
+
+def make_report():
```

Comment:
Added `CLI` utility that can reproduce the latest `html` report.

Reply:
This is very cool and we may need it at some point, but I think it might be overkill in this PR. Does it add enough value yet over documenting `report.html_report()` and keeping everything in the Python API?

Reply:
> Does it add enough value yet over documenting report.html_report() and keeping everything in the Python API?

Probably it doesn't.

Reply:
Removed